lib/Video.js

```diff
@@ -550,6 +551,7 @@ module.exports = kind(
 
 		this.setPlaybackRate(this.selectPlaybackRate(this._speedIndex));
 
+		isNeedPlay && node.play();
```

Comment:
Not a huge fan of these types of statements (but open for debate). I think it'd be clearer and not many characters to do something like this:
```
if (isNeedPlay) node.play();
```